akka-http-core/src/main/scala/akka/http/javadsl/settings/SettingsCompanion.scala

```diff
@@ -10,7 +10,7 @@ trait SettingsCompanion[T] {
    *
    * Java API
    */
-  final def create(system: ActorSystem): T = create(system.settings.config)
+  def create(system: ActorSystem): T = create(system.settings.config)
```

Comment:
Yeah that'd be good. I can do that tomorrow and we'll just merge this :) Want to have it in next release.